.github/workflows/ci.yml

```diff
@@ -800,8 +800,8 @@ jobs:
       HOST_BUILD_DIR: ${{ needs.build_all.outputs.build-dir }}
       HOST_BUILD_DIR_ARCHIVE: ${{ needs.build_all.outputs.build-dir-archive }}
       HOST_BUILD_DIR_GCS_ARTIFACT: ${{ needs.build_all.outputs.build-dir-gcs-artifact }}
-      BENCHMARKS_ARCHIVE: ${{ needs.benchmarks.outputs.benchmarks-archive }}
-      BENCHMARKS_GCS_ARTIFACT: ${{ needs.benchmarks.outputs.benchmarks-gcs-artifact }}
```

Comment:
Can clean up benchmark.outputs.benchmark-gcs-artifact et. al in https://github.com/iree-org/iree/blob/f7a12afba4013d5ef168f09055118697d4147793/.github/workflows/benchmarks.yml#L40-L49

and

https://github.com/iree-org/iree/blob/f7a12afba4013d5ef168f09055118697d4147793/.github/workflows/benchmarks.yml#L103-L124 to save GCS space since the tarball is not used.

Reply:
Done. Only build the legacy benchmark suite for the testing purpose now. The whole job should be removed once we deprecate the legacy benchmark suite.